examples/avmnist_multimodal/README.md

```diff
@@ -0,0 +1,40 @@
+# Multimodal Digit Classification: Multimodal Deep Learning (MMDL) Method on AVMNIST
+
+### 1. Description
+
+This example is constructed by refactoring the code of [MultiBench repository](https://github.com/pliang279/MultiBench) for [Pykale library](https://github.com/pykale/pykale).
```

Comment:
missing "the": the MultiBench ..., the PyKale library

Reply:
fixed